### ISSUE TITLE
[C#] Syntax Highlighting not working fully

### ISSUE THREAD

**Issue by cable729** [Microsoft/vscode#1683]
For the latest version of vscode (0.10.6), my syntax highlighting for C# is broken. I don't know if this is a temporary file issue, or something else. I tried reinstalling vscode, but it didn't help.

![image](https://cloud.githubusercontent.com/assets/1524578/12027895/79e350be-ad8b-11e5-904d-1f1af47311fc.png)

I would expect `ICloneable` to be highlighted, as well as `Class1`, `MyEnum`, and the `One` on line 13. As you can see, they're not highlighted at all. When I inspect `ICloneable`, I get this:

```html
<span class="token body class meta method namespace property return-type cs">ICloneable&nbsp;</span>
```

I'm not sure what is causing this, but maybe there's some cache files that I can clear. If so, where are those located?

Thanks,
Caleb

# **Reproduction**

1. Create a new class library.
2. Write some silly code.
3. Load it in VsCode.
4. It's not highlighted correctly.

**Comment by egamma** [comment]
CC @aeschli

**Comment by aeschli** [comment]
In the December we modified the default white theme to be in sync with the original Visual Studio light and dark themes. So what you see is expected. You can follow the discussion in #1270. The colors will be available in new themes, see #1849
